TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -1,23 +1,40 @@
 version: '3.7'
 services:
 
-  captureproxy:
+
+  # Run combined instance of Capture Proxy and Elasticsearch
+  capture_proxy_es:
```

Comment:
An "_and" thrown in here will help drive home that you're doing both (I didn't realize it until I saw the pair of containers commented out)